### ISSUE TITLE
Ability to submit issue report separately

### ISSUE THREAD

**Issue by datherton09** [responsible-ai-collaborative/aiid#2739]
@kepae Feature request: A quick ability to switch the submission form from an incident report to an issue report. Even if this is something that is only available to editors and not the general public, that would be really useful.

Currently, I spend a lot of time filling out the incident report form in full for reports I submit as an issue and not as an incident. Looking at the issue reports, it seems like none of the information I fill out makes it to published form, so it's essentially (seemingly?) wasted effort. But also just from a terminology standpoint, it seems like we need a basic issue report form.

If the information that gets filled out in the incident report form for an issue report does get saved and stored, I can keep doing that, but it'd be handy to have a way to quickly add issue reports without writing about them as if they are incidents (only to have the information seem to disappear).

**Comment by clari182** [comment]
<img width="1597" alt="Screenshot 2024-04-24 at 12 36 31" src="https://github.com/responsible-ai-collaborative/aiid/assets/4105289/79fb9407-f5d0-4435-accd-853666c1b33d">
Maybe one way would be to update the form if the editor chooses "Issue" inside the dropdown.

Do you know the fields that are not visible on an Issue Report?

**Comment by datherton09** [comment]
@clari182 That would be great! It seems that the only fields visible after issue reports get published on the site are the title, text of the report, and publication date.

**Comment by clari182** [comment]
@datherton09 @kepae 
Actually I think all the data filled in the submission is necessary for the issue report. It's not currently visible but I think it should be filled. When you're seeing an issue report like https://incidentdatabase.ai/reports/2974/ and you edit it, all fields are displayed.

Maybe the issue is that we are not displaying all the fields on the report page as we should. What do you think?

**Comment by kepae** [comment]
That sounds ideal to me. All data about reports should be visible to editors. What data is specifically stored but not presented in the report view?

**Comment by datherton09** [comment]
@clari182 @kepae Good point! I'm wondering then if in a revised issue report submission form that we only need to change these fields:

- incident date -> remove this because the report's date of publication stands as the date
- alleged deployer -> potentially include something like an "N/A" check box because there may be issue reports that do not discuss someone deploying the tech that ultimately harms someone (as things currently stand, the fields must be filled out in order to submit for ingestion)
- alleged developer -> ditto the inclusion of a check box in case the issue report contends with some sort of issue that transcends any specific developer
- alleged harmed parties -> maybe change this to "Potential risk of harm" or "Potential parties who may be harmed"? Or "identified risk"? Something along those lines? (As issue reports are future-oriented.)

**Comment by clari182** [comment]
@kepae @datherton09  what we display for editors (and everyone) in reports is the same for incident reports and issue reports and it's the following:
- Report number
- Classifications if any
- Report image
- Title
- URL
- Translation badge (if you're viewing the report in a different language than its original)
- Report text
- Variants if any
- Authors
- Submitters
- Report's history

When an editor is logged in it displays an `Edit` button which lets you view all the other fields.

Regarding renaming the fields, I think what is happening here is that `Incident reports` also creates an Incident while `Issue Reports` don't (correct me if I'm wrong). So that whole section is not really necessary for `Issue Reports`.
Maybe what we need to do instead of renaming is adding those fields `alleged developer`, `alleged deployer` and `harmed parties` as new reports fields _only_ for issue reports. Because right now, they are mandatory but later ignored if it's an `Issue Report`.

Feel free to open a slack chat to discuss this further so we can work out the details on what needs to be done!
